scheduler-driver/src/main/java/gov/nasa/jpl/aerie/scheduler/simulation/SimulationFacade.java

```diff
@@ -41,22 +39,30 @@ public class SimulationFacade implements AutoCloseable{
   private ResumableSimulationDriver<?> driver;
   private int itSimActivityId;
 
-  //simulation results from the last simulation, as output directly by simulation driver
-  private SimulationResults lastSimDriverResults;
-  private gov.nasa.jpl.aerie.constraints.model.SimulationResults lastSimConstraintResults;
   private final Map<SchedulingActivityDirectiveId, ActivityDirectiveId>
       planActDirectiveIdToSimulationActivityDirectiveId = new HashMap<>();
   private final Map<SchedulingActivityDirective, ActivityDirective> insertedActivities;
-  private static final Duration MARGIN = Duration.of(5, MICROSECONDS);
   //counts the total number of simulation restarts, used as performance metric in the scheduler
   private int pastSimulationRestarts;
 
-  public gov.nasa.jpl.aerie.constraints.model.SimulationResults getLatestConstraintSimulationResults(){
-    return lastSimConstraintResults;
+  public SimulationData lastSimulationData;
+
+  /**
+   * The set of activities to be added to the first simulation.
+   * Used to potentially delay the first simulation until the loaded results are stale.
+   * The only way to add activities to the facade is to simulate them. But sometimes, we have initial sim results and we
+   * do not need to simulate before the first activity insertion. This initial plan allows to "load" the activities in simulation
```

Comment:
Minor: typo (I'm assuming `the facade` is the dropped subject?)
```suggestion
   * do not need to simulate before the first activity insertion. This initial plan allows the facade to "load" the activities in simulation
```